src/Core/Scheduler/Providers/$3dTiles_Provider.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Created On: 2016-11-8
+ * Class: $3dTiles_Provider
+ * Description:
+ */
+import * as THREE from 'three';
+import Provider from './Provider';
+import B3dmLoader from '../../../Renderer/ThreeExtended/B3dmLoader';
+import Fetcher from './Fetcher';
+import BasicMaterial from '../../../Renderer/BasicMaterial';
+
+
+export function $3dTilesIndex(tileset, urlPrefix) {
```

Comment:
The name `$3dTilesIndex` is fine but I'd suggest renaming the files and remove their `$` prefix.